code/UI/General/TabMenus/Scoreboard/Entry/ScoreboardEntry.cs

```diff
@@ -45,4 +77,28 @@ public void Update()
 
 		PlayerAvatar.SetTexture( $"avatar:{_client.PlayerId}" );
 	}
+
+	public void OnClick()
+	{
+		// Only allow tagging currently alive players, and only if the selected player is not the local player
+		if ((_client.Pawn as Player).Status is not PlayerStatus.Alive || _client.Pawn.IsLocalPawn)
+			return;
+
+		if (!Expanded) {
+			Style.Set("height", "76px");
+			GetChild(1).RemoveClass("no-display");
+			Expanded = true;
```

Comment:
We don't need the `Expanded`, we could just check if the panel is visible or not.